src/test/java/controllers/ProjectInputControllerTest.java

```diff
@@ -144,23 +194,23 @@ void testProjectViewCredits() {
     }
 
     @Test
-    void testProjectAddTask() {
+    void testProjectAddTask_valid() {
         try {
             Project project = new Project("Infinity_Gauntlet");
             dueDate = dateTimeHelper.formatDate("21/09/2019");
             simulatedUserInput = "add task -t Documentation for product -p 2 -d 21/09/2019 -c 40 -s todo "
                     + "-r do something -r do another thing";
             projectInputController.projectAddTask(project, simulatedUserInput);
 
```

Comment:
Simply changed the task name here because I edited it such that tasks with the same name cannot be added.

Reply:
i am not sure i understand the reason for this change. Does it mean only positive test cases can be under that method?

Reply:
Yup. I think so. I'm trying to follow the format seen in the notes as shown here:
![image](https://user-images.githubusercontent.com/42560942/68089924-c84d9800-fea8-11e9-9332-6f3fc0e145d8.png)
Here's the associated page in the textbook
https://nuscs2113-ay1920s1.github.io/website/se-book-adapted/chapters/junit.html#junit

Reply:
alrite got it